workflow/exp/exp.hera_conus12km

```diff
@@ -18,8 +18,12 @@ export REALTIME=true     # true, false
 export PHYSICS_SUITE="convection_permitting"   # mesoscale_reference, convection_permitting
 export ZETA_LEVELS="L60.txt"
 #
-export FCST_ONLY="false"
-export CYC_INTERVAL=3
+export DA_IODA="true"
+export DA_JEDI="true"
+export OBSPATH="/public/data/grids/rap/obs"
```

Comment:
It looks like your PR is NOT based on the latest branch? I think lines 21-23 have been merged in a previous PR.

Reply:
Last PR is only for jet conus 12km configure. This is for Here.

Reply:
Got it